packages/components/src/components/logos/yoga.tsx

```diff
@@ -11,48 +11,19 @@ export const YogaLogo = ({ className }: { className?: string }): ReactElement =>
   >
     <path
       d="M9.48276 21.584C10.8633 21.584 12.1749 21.8798 13.358 22.4116L49.5832 1.52142C51.2901 0.570589 53.3763 0 55.2728 0C57.359 0 59.2556 0.570589 60.9625 1.52142L92.066 19.5871C89.7901 21.1085 88.0832 23.3903 86.9453 26.0526L56.7901 8.55744C56.2212 8.17721 55.6525 8.17721 55.0836 8.17721C54.5147 8.17721 53.9453 8.36728 53.3763 8.55744L18.629 28.5725C18.8484 29.375 18.9655 30.2199 18.9655 31.0922C18.9655 36.3435 14.7199 40.6005 9.48276 40.6005C4.24558 40.6005 0 36.3435 0 31.0922C0 25.841 4.24558 21.584 9.48276 21.584Z"
-      fill="url(#paint0_linear_2_28)"
+      fill="#c026d3"
```

Comment:
I guess `fill` can be removed here and applied directly to `<svg/>` element, so we could override it with class in the future

Reply:
@enisdenjo pushed a fix, for me it's fine, let's wait feedback for others 
<img width="403" alt="image" src="https://user-images.githubusercontent.com/7361780/196208310-411cbb3a-b983-4c08-89e9-199bcb81b740.png">